projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -1,24 +1,28 @@
 ## Global Docker image parameters
 ## Please, note that this will override the image parameters, including dependencies, configured to use the global value
-## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+## Current available global Docker image parameter(s): imageRegistry
 ## Set global.imageRegistry if you use docker proxy to the proxy prefix for versatiledatakit.
 ##
 global: {}
 #   imageRegistry: myRegistryName
-#   imagePullSecrets: # TODO enable setting imagePullSecrets so that clients can use any docker repos easier (clients can still set it up manually)
 #     - myRegistryKeySecretName
 
 # Image configuration of the Control service image. Generally those should be left as it is.
 image:
-   registry: registry.hub.docker.com/versatiledatakit
-   repository: pipelines-control-service
-   tag: "1f57f0e"
-
-   ## Specify a imagePullPolicy
-   ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
-   ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
-   ##
-   pullPolicy: IfNotPresent
+  registry: registry.hub.docker.com/versatiledatakit
+  repository: pipelines-control-service
+  tag: "1f57f0e"
+  ## Image registry types. Your account need to have pull permissions over the registry. The supported values are:
+  ## - "generic" : Dockerhub or Harbor type registries
+  ## TODO Support will be added for more later
+  registryType: generic
```

Comment:
We are unlikely to support other types. 

The way it's handled is we'd allow for users to specify their own imagepull secrets in the future (instead of us generating one in the helm chart) if user/pass is not enough. 

So I'd vote to remove registryType. 

You probably saw registryType for builder configuration. But there we push images so we need to support pushing to ECR or Google ,etc which may be different (usually creating repository is different)